### ISSUE TITLE
fatal error on readability check

### ISSUE THREAD

**Issue by norcross** [equalizedigital/accessibility-checker#87]
we are seeing the following error all over the logs:

`PHP message: Fatal error: Uncaught TypeError: Cannot access offset of type string on string in /var/www/wp-content/plugins/accessibility-checker/accessibility-checker.php:1222`

in taking a look, the post meta is being pulled but there is no confirmation that any of the data exists before beginning to process it.

**Comment by SteveJonesDev** [comment]
The post meta is likely missing because of the server access restrictions. But it's good practice to check for the array index so it fails more gracefully. This has been updated with the 1.3.7 release that I just pushed up to the plugin repo.
